### ISSUE TITLE
Clarification to Databases using SQL lesson

### ISSUE THREAD

**Issue by ilmcconnell** [datacarpentry/sql-ecology-lesson#287]
In the 'Questions' section the text states:

> First, let’s download and look at some of the cleaned spreadsheets from the Portal Project dataset.
> We’ll need the following three files:

The data was already downloaded in the setup section, and the learners are again reminded to do that at the beginning on of the lesson. I would expect this text to reference the already downloaded files like so:

> First, let’s look at some of the cleaned spreadsheets you downloaded (during setup) from the Portal Project dataset.
> We’ll need the following three files:

**Comment by MargoLawrence** [comment]
The lesson is designed to get the learner up to date with the data in the files.  The challenge states that they want them to find what information is in each file, and how they would be answered.  The goal of the lesson is to understand the strength of a database vs a dataset in a spreadsheet, this can only be accomplished by trying to answer the questions in the challenge.

**Comment by ilmcconnell** [comment]
@MargoLawrence I completely agree. I’m only trying to point out that the learners don’t need to download the files a second time.

**Comment by ChristinaLK** [comment]
Hi @ilmcconnell ! :)

thanks for pointing this out. it's likely there because we want people to have the right data (which they may have lost / not downloaded correctly / who knows since the last lesson), but I think we could clarify to say either - use these files from before or download them again if you don't have them for some reason.

**Comment by dvdan-jgm** [comment]
@ChristinaLK I will take this as my checkout contribution.
